src/test/java/formflow/library/file/UploadJourneyTests.java

```diff
@@ -60,7 +60,7 @@ void documentUploadFlow() {
     //Race condition caused by uploadPasswordProtectedPdf waits until upload file has file details added instead
     //of waiting until file upload is complete.
     try {
-      Thread.sleep(500);
+      Thread.sleep(2000);
```

Comment:
oh no, we're going to wait 2 seconds now? lol

Reply:
For some reason it kept faling at 500 and 1000 :/